pkg/api/api.go

```diff
@@ -64,6 +64,7 @@ var (
 )
 
 func (api *API) Register(mux *http.ServeMux) {
+	mux.HandleFunc("healthz", ok)
```

Comment:
did you mean this to be `/healthz` ?

Reply:
Was going off the convention of using healthz in other projects. Is there a better convention to use?

Reply:
I was just wondering if the preceding `/` is necessary or not